proposals/2024-06-14_mixxx_e_v.md

```diff
@@ -0,0 +1,567 @@
+   may be held exclusively by these means.
+
+## 7. Duties of the General Assembly
+
+1. The General Assembly is the highest decision-making organ and is
+   responsible for all duties unless otherwise assigned.
+2. It elects the Board from a number of core members and can deselect board
+   members.
+3. It decides on member expulsion objections.
+4. It receives the Board’s annual report and auditor’s report, and discharges
+   the Board’s responsibilities.
+5. It decides on amendments to the Articles of Association and liquidation of
+   the Association.
+6. Namely the annual accounts and the annual report have to be presented in
+   written format to the General Assembly for decision making and discharge
+   of the board. It calls on an auditor, who belong neither to the board nor
+   any other panel appointed by the board and who is not employed by the
+   association, in order to audit the accounting and annual report and
+   relate their results to the General Assembly. The auditors have access
+   to all of the Association’s accounting documents.
+7. It decides on purchases, sales, investments, and raising loans.
+
+## 8. The Board
+
+1. The Board consists of the Chairman, the Second Chairman, and the Treasurer
+   elected for a 3-year term.
+2. The Board’s duties and meeting procedures are stipulated in the by-laws.
+   The Chairman is responsible for following the purpose of the association,
+   complying with the Articles of Association and delegation of tasks. The
+   second chairperson represents the Chairman in his absence and verifies the
+   resolutions of the Board and the General Assembly. The treasurer manages
+   the finances of the association, keeps the cash books and the list of
+   members.
+3. The Board decides on all matters unless a General Assembly decision is
+   required. It implements the decisions taken by the General Assembly.
+4. Each Board member has the right to represent the Association.
+5. Should a board member quit while in office, the board appoints a
+   provisional board member. The provisional board member remains in
+   office until the next General Assembly takes place. The General
+   Assembly decides about the definite succession within the board.
+6. The Board can appoint a director to manage current affairs.
+7. The director participates in General Assemblies and Board meetings.
+8. The Board can amend the Articles of Association for formal reasons as
+   required by authorities. The amendment will be voted on at the following
+   General Assembly.
+
+## 9. By-Laws
+
+The by-laws govern all details of the Association’s activities not covered in
+the Articles of Association. The Board implements the by-laws following
+core member approval by internet vote.
+
+## 10. Meeting Minutes
+
+All decisions by the Board and General Assembly are documented in an 
+electronically translatable format in German and available for members’ 
+perusal and for the examination of the resolutions by the registry court.
```

Comment:
The certification of the resolutions is missing in accordance with § 58 No. 4 BGB. Who signs the minutes of the general meeting?